docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Security considerations
+parent: Operating System
+nav_order: 330
+---
+## Security considerations
+
+As a networked device, reachable from the outside, the attack surface of the BitBox Base needs to be minimized as much as possible. This is why we use our BitBox App as the dedicated user interface for node management. By default, only one port is exposed on the Base to communicate over an end-to-end encrypted channel with the App, providing all functionality.
+
+### Pairing Base with BitBox App
+The first step is to safely pair the BitBox Base and App. To achieve this, the Base announces itself on the local network using mDNS and the Middlware exposes provides an API endpoint to initiate a secure connection. The Base is automatically detected by the App and announced within the user interface. 
+
+At first, only a single public API endpoint for pairing is available that allows the two components to establish an encrypted communication channel. To rule out a man-in-the-middle attack, a confirmation code is shown both within the App and on the Base and needs to be confirmed manually. With the secure connection established, the [Noise encryption](http://noiseprotocol.org) can be set up and the full API functionality is available to the App.
```

Comment:
(Optional) Up to you how to approach this, @Stadicus, but do you think it would make sense to either hold off on documenting the mutually encrypted/authenticated communication channel between App -> Base until we've implemented it, or to add some note in the docs on the current status of the features described here (e.g link to a publicly reachable issue)?

Reply:
This very high-level description is how I understood our plans to implement this currently. Is your concern that it's a) wrong, or b) not specific enough?

Reply:
> Is your concern that it's a) wrong, or b) not specific enough?

My question was:
> do you think it would make sense to either hold off on documenting the mutually encrypted/authenticated communication channel between App -> Base until we've implemented it, or to add some note in the docs on the current status of the features described here (e.g link to a publicly reachable issue)?

So what I was asking is how to handle documentation of not yet implemented features; with two suggested options:

1. Document at the time we implement the feature
2. Document now, but make clear it's describing a future state, and link to publicly viewable issue that onlookers can follow in order to see progress

Reply:
I'd opt for 2) but without publicly viewable issue, unless there are different reasons to do so. Nothing wrong with outlining our goals, even if not implemented yet, IMO.

Reply:
The main risk that I see is that we'll end up with documentation that is too optimistic relative to what we can actually achieve, which later ends up being an obstacle more than an asset when the technical artifacts have diverged significantly from the documentation. (This seems to have happened with a lot of docs around BitBox02.)

Without any publicly viewable issues, how do you suggest we'll mark documentation describing intentions and not (yet) reality (i.e what do we link the interested user / possible contributor to?), and how will we remember to revisit that documentation once we've implemented the features we're describing?

Reply:
Let's discuss with @TheCharlatan how he want's to handle this, as he's the one implementing this feature.

Reply:
> Let's discuss with @TheCharlatan how he want's to handle this

Sure, we can do that, although the question is more wide than just for this specific feature. See e.g the other comment for `security.md` on L27, where I ask the same question around the feature to add SSH public keys via the App.

Reply:
We decided to add a comment in the documentation, linking to our internal issue tracking the development.